theseus_gui/src/pages/project/Versions.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <Card>
+    <div class="filter-header">
+      <div class="manage">
+        <DropdownSelect id="filterLoader" placeholder="Filter loader..." :options="versions.flatMap(value => value.loaders).filter((value, index, self) => self.indexOf(value) === index)" class="select no-wrap"/>
+        <DropdownSelect id="filterVersions" placeholder="Filter versions..."  :options="versions.flatMap(value => value.game_versions).filter((value, index, self) => self.indexOf(value) === index)" class="select no-wrap"/>
+      </div>
+      <div class="manage">
+        <Checkbox id="filterCompatible" label="Only show compatible versions" />
+        <Button class="no-wrap">
+          <CheckCircleIcon />
+          Clear Filters
+        </Button>
+      </div>
+    </div>
+  </Card>
+  <Card class="mod-card">
+    <div class="table-container">
```

Comment:
This design is alright but lacks a lot of features from the main site. The entire row should be clickable and there should be a hover state on the row. Why not just use a similar style to the main site?